contracts/IPRBProxy.sol

```diff
@@ -2,11 +2,12 @@
 pragma solidity >=0.8.4;
 
 import { IERC1155Receiver } from "@openzeppelin/contracts/token/ERC1155/IERC1155Receiver.sol";
+import { IERC721Receiver } from "@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol";
 
 /// @title IPRBProxy
 /// @author Paul Razvan Berg
 /// @notice Proxy contract to compose transactions on owner's behalf.
-interface IPRBProxy is IERC1155Receiver {
+interface IPRBProxy is IERC1155Receiver, IERC721Receiver {
```

Comment:
does it inherit the ERC165 implicitly?

Reply:
or does it even need to be ERC165 compliant?

Reply:
It seems that the ERC165 is needed for the [ERC1155 receiver](https://eips.ethereum.org/EIPS/eip-1155#erc-1155-token-receiver) but not for the [ERC721 receiver](https://eips.ethereum.org/EIPS/eip-721#specification). I guess that's why OZ implementation includes it in the ERC1155Holder but not on the ERC721Holder